pyproject.toml

```diff
@@ -4,21 +4,27 @@ build-backend = "poetry.masonry.api"
 
 [tool.poetry]
 name = "django-safemigrate"
-version = "3.0"
+version = "4.0"
```

Comment:
No need for a major version bump if the release is compatible. When you drop support for older Python or Django versions you’ll need to increment the major version, but I don’t see any breaking changes in this PR atm.

Reply:
Yeah, this might have been an artifact of earlier thinking. Good catch.